juju/client/facade.py

```diff
@@ -286,6 +294,20 @@ def as_kwargs(self):
             return ', '.join(parts)
         return ''
 
+    def as_validation(self):
+        if self:
```

Comment:
why the if self? is this called as a static method somewhere?

Reply:
that I don't know, this is the same code as `as_kwargs` and I had assumed it was done for a reason, if that's not the case, I can change that.

Reply:
self is the instance of the class so if this were to be used like a static method I could see self not being defined. It's odd to see something that could be used in this way though.